config/setup.js.tpl

```diff
@@ -1,30 +0,0 @@
-function define(name, value) {
```

Comment:
Why deleted?

Reply:
Most likely because I renamed it. I'll address.